README.md

```diff
@@ -4,95 +4,53 @@
 
 > InterPlanetary Linked Data
 
-Read more about IPLD at https://ipld.io
+Captain: [@nicola](https://github.com/nicola)
 
-_Note that, as in many IPFS repositories, most of the work is happening in [the issues](https://github.com/ipld/ipld/issues/) or in [active pull requests](https://github.com/ipld/ipld/pulls/). Go take a look!_
 
-### IPLD Definitions
-
-There are a variety of systems that use merkle-tree and hash-chain inspired datastructures (e.g. git, bittorrent, IPFS, tahoe-lafs, sfsro). IPLD defines:
-
-- merkle-links: the core unit of a merkle-graph
-- merkle-dag: any graphs whose edges are merkle-links.
-- merkle-paths: unix-style paths for traversing merkle-dags with named merkle-links
-- IPLD Data Model: a flexible, JSON-based data model for representing merkle-dags.
-- IPLD Serialized Formats: a set of formats in which IPLD objects can be represented, for example JSON, CBOR, CSON, YAML, Protobuf, XML, RDF, etc.
-- IPLD Canonical Format: a deterministic description on a serialized format that ensures the same logical object is always serialized to the exact same sequence of bits. This is critical for merkle-linking, and all cryptographic applications.
-
-In short: JSON documents with named merkle-links that can be traversed.
-
-## Table of Contents
-
-- [Repositories](#repositories)
-  - [Specifications](#specifications)
-  - [Implementations](#implementations)
-    - [New implementations](#new-implementations)
-  - [Other Tools](#other-tools)
-  - [Meta repos](#meta-repos)
-- [Lead](#lead)
+- [Overview](#overview)
+- [New implementations](#new-implementations)
 - [Contribute](#contribute)
 - [License](#license)
 
-## Repositories
-
-IPLD is a multifaceted, distributed effort.
-
-### Specifications
-
-- [specs](https://github.com/ipld/specs) - The specifications for IPLD
-- [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
-
-### Implementations
-
-IPLD has multiple types of implementations, some of which have been written in multiple languages.
-
-#### Data format descriptions
-
-- [js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
-- [js-ipld-dag-pb](https://github.com/ipld/js-ipld-dag-pb) - JavaScript Implementation of the MerkleDAG Nodes with Protobuf.
-- [js-ipld-eth-block](https://github.com/ipld/js-ipld-eth-block) - JavaScript Implementation of the IPLD format - Ethereum Block
-- [go-ipld-btc](https://github.com/ipfs/go-ipld-btc)
-- [go-ipld-zcash](https://github.com/ipfs/go-ipld-zcash)
-- [go-ipld-cbor](https://github.com/ipfs/go-ipld-cbor)
-- [go-ipld-git](https://github.com/ipfs/go-ipld-git)
-- [go-ipld-node](https://github.com/ipfs/go-ipld-node)
-
-#### Interface definitions
 
-- [interface-ipld-format](https://github.com/ipld/interface-ipld-format) - A interface you can follow to implement a valid IPLD format, resolvable through the IPLD Resolver (available in IPFS)
+## Overview
+
```

Comment:
We should probably add a blurb here explaining *WTF* is IPLD. Honestly, we can just steal it from the website:

> IPLD is the data model of the content-addressable web. It allows us to treat all hash-linked data structures as subsets of a unified information space, unifying all data models that link data with hashes as instances of IPLD.